get/build.go

```diff
@@ -84,7 +88,12 @@ func printBuild(builds []apps.Build, get *Cmd, out io.Writer, header bool) error
 }
 
 func pullImage(ctx context.Context, apiClient *api.Client, build *apps.Build) error {
-	cli, err := client.NewClientWithOpts(client.FromEnv)
+	value, exists := os.LookupEnv("DOCKER_API_VERSION")
```

Comment:
I think you actually don't need that. `client.WithVersion(dockerAPIVersion)` will set the hardcoded docker API version to 1.42. If the user specified an environment variable `DOCKER_API_VERSION`, then `client.FromEnv` should pick this up and overwrite our default version.